bundle-config-loader.ts

```diff
@@ -4,7 +4,7 @@ import { getOptions } from "loader-utils";
 import * as escapeRegExp from "escape-string-regexp";
 
 // Matches all source, markup and style files that are not in App_Resources
-const defaultMatch = "(?<!App_Resources.*)\.(xml|css|js|(?<!d\.)ts|scss)$";
+const defaultMatch = "(?<!App_Resources.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!_[\\w-]*\\.)scss)$";
```

Comment:
This RegExp will not match folders containing App_Resources in their name:
App_Resources123, MyApp_Resources, etc. Is this expected?
If not, you can change the RegExp to:
```JavaScript
const defaultMatch = "(?<!\\bApp_Resources\\b.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!_[\\w-]*\\.)scss)$";
```

Reply:
If I understand this RegExp correctly, we do not allow having `_` in the name of the scss file, i.e.
`app_component.scss` will not match. Am I right? Is this the intended behavior?

Reply:
If we want to allow having `_` in the names of the `scss` files, we can add `\\b` in the negative lookbehind statement. Combining the RegExp with the suggestion from my previous comment, it would look like:
```JavaScript
const defaultMatch = "(?<!\\bApp_Resources\\b.*)\\.(xml|css|js|(?<!\\.d\\.)ts|(?<!\\b_[\\w-]*\\.)scss)$"
```

Reply:
I've tried to test the regexp (the suggestion from my comment) and I've noticed the following file still matches the RegExp `_random'.scss"`. But... I do not expect anyone to have such file.

Reply:
Yup. Currently using `[\w-]*` to match the file name. 
Also `_patial;drop-table-users.scss` is a potential SQL injection vulnerability 🙊